MonoGame.Framework/Game.cs

```diff
@@ -19,6 +19,10 @@ namespace Microsoft.Xna.Framework
 {
     public class Game : IDisposable
     {
+#if DESKTOPGL
+        public static bool Resizable = false;

```

Comment:
> OpenTK does not support it with SDL backend,

I'm taking about OpenTK ... not OpenTK with SDL.  Did OpenTK support changing the window to resizable after creation?